graphql/task_resolver.go

```diff
@@ -366,7 +366,7 @@ func (r *taskResolver) ExecutionTasksFull(ctx context.Context, obj *restModel.AP
 func (r *taskResolver) FailedTestCount(ctx context.Context, obj *restModel.APITask) (int, error) {
 	dbTask, err := obj.ToService()
 	if err != nil {
-		return 0, InternalServerError.Send(ctx, fmt.Sprintf("Error getting service model for APITask %s: %s", *obj.Id, err.Error()))
+		return 0, InternalServerError.Send(ctx, fmt.Sprintf("xgetting service model for APITask %s: %s", *obj.Id, err.Error()))
```

Comment:
`utility.FromStringPtr`
```suggestion
		return 0, InternalServerError.Send(ctx, fmt.Sprintf("getting service model for APITask '%s': %s", *obj.Id, err.Error()))
```